### PR TITLE
wechaty-electron blog

### PR DESCRIPTION
**Pull request by GuoYingxu**
wechaty-electron 如何让你的wechaty 变成客户端

Fixes # .

Make sure these boxes are checked before creating your pull request -- thank you!

- [x] Add Blog Header, including title, author, date.
e.g.
```
---
 layout: post
 title: "'Score Your Face Photo' a ML&Wechaty practice"
 date: 2017-09-18 09:00 +0800
 author: huyingxi
 ---
 
 > Author: [@huyingxi](https://github.com/huyingxi/wechaty_selfie) enjoying ML&Wechaty at BUPT
```
- [x] Keep all filenames & url as lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor` instead of `2017-10-06-WeChat PC Impactor`,
- [x] Embed the photo & video before publishing, save all external file to the blog `/download/2017` directory.
- [x] Add `<!-- more -->` section to just show the abstract in blog homepage.

[See more](https://github.com/Chatie/blog#contribute-guideline)

### CODE REVIEW

File: _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "wechaty-electron 把你的wechaty 变成客户端"
+date: 2017-11-26 12:00 +0800
+author: 郭英旭
+---
+> 作者: [郭英旭](https://github.com/Guoyingxu)
+
+## 初识wechaty
+去年的这个时候，我刚到现在的公司任职，发现同事正在用微信做转播，就是把几个微信群里有价值的发言同步到其他几个群里，同时还要存入数据库以便在论坛上也能看到相关讨论。当时同事用的是chrome插件的方式，那是第一次知道微信还可以这么玩。惊奇的同时我也发现了一个很严重的问题，就是插件跑一段时间，网页就卡死了，就得重启。 于是我在网上找不同的解决方案，github 真是一个神奇的地方，很快我就找到了好几个版本的微信机器人，人气最高的当属wechaty，另外还有一个比较看好的是一个python的项目。由于本人对nodejs 对偏爱，选择了wechaty。
```
Comment (lijiarui) [inline _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md]: Could you please add `<!-- more -->` section here or anywhere you like to just show the abstract on blog homepage.

**Review by lijiarui — CHANGES_REQUESTED** [review]

**Comment by lijiarui** [comment]
Thanks for your pr!

Also， here are 2 suggestion:

1. Could you change the file-name from `2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md` to something like `2017－11-26-wechaty-electron-making-your-wechaty-as-a-client-service.md`, because wechaty blog keeps all filenames & url as lowercase, and use - to connect words instead of space.

2. I suggest to add a image here to make the blog more attractive.

Thanks!

File: _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  app.quit()
+}) 
+function initbot(){
+  //这里与正常使用wechaty代码一样
+}
+```
+   
+    
+参考 readme quick start 和  [wechaty-electron-example](https://github.com/GuoYingxu/wechaty/tree/wechaty-electron/example/electron-app) 
+
```
Comment (lijiarui) [inline _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md]: Also I suggest to add [wechaty-electron](https://github.com/GuoYingxu/wechaty/tree/wechaty-electron) quick start method here


1. Go to repo: https://github.com/GuoYingxu/wechaty/tree/wechaty-electron
2. Run the following command
```
  git checkout wechaty-electron
  npm install
  npm run dist
  npm start
```

**Review by lijiarui — CHANGES_REQUESTED** [review]
I just to let others know how to run in this blog.

I just copy your readme here, can try the way you like, thanks!

File: _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md (hunk 2 above)
Comment (GuoYingxu) [reply _posts/2017－11-26-wechaty-electron 如何让你的wechaty 变成客户端.md]: @lijiarui  今天弄一些图片。然后再录个小视频。晚上回去重新修改一遍。

File: _posts/2017-11-26-wechaty-electron-making-your-wechaty-as-a-client-service.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+---
+layout: post
+title: "wechaty-electron 把你的wechaty 变成客户端"
+date: 2017-11-26 12:00 +0800
+author: 郭英旭
+---
+> 作者: [郭英旭](https://github.com/Guoyingxu)
+
```
Comment (lijiarui) [inline _posts/2017-11-26-wechaty-electron-making-your-wechaty-as-a-client-service.md:8]: I suggest to add one of your image before `more` to make it more attracitve

**Review by lijiarui — CHANGES_REQUESTED** [review]
Thanks！ 

Will you add video to this blog as you said yesterday? If you just keep this version without video, I will merge when you add an image to your blog's title. Thanks!

See http://blog.chatie.io/

**Comment by GuoYingxu** [comment]
@lijiarui 我添加了几个图片，视频就算了

**Pull request closed without merging**